t/ffi_platypus_lang.t

```diff
@@ -42,11 +41,10 @@ subtest 'Foo attribute' => sub {
   is $ffi->sizeof('foo_t'), 2, 'sizeof foo_t = 2';
   is $ffi->sizeof('bar_t'), 4, 'sizeof foo_t = 4';
 
-  is $ffi->function('UnMangled::Name(int i)' => ['myint'] => 'myint')->call(22), 22;
+  is $ffi->function('UnMangled::Name(int i)' => ['bmyint'] => 'bmyint')->call(22), 22;
```

Comment:
I don't understand how I didn't run into this earlier.  this mangles to `f0` which takes a `uint8` not a `sint32`.